### ISSUE TITLE
Logo RSI (Senegal), which font?

### ISSUE THREAD

**Issue by Lexzie** [picons/picons-source#313]
As you can see, this logo is quite simple.
![afbeelding](https://user-images.githubusercontent.com/14023349/35975488-3cdab956-0cdd-11e8-8ef8-1da39cf51d45.png)

But the ones I could find were of poor quality.
Can anyone identify the fonts that are used? I could try to make the logo in Inkscape.
Or if anyone else feels like doing that?

**Comment by mcfly822** [comment]
It was not so hard to find out... I tried first Arial, and done 😸 
Exactly: Arial Heavy.
See here:
[rsi.default.svg_mcfly822.zip](https://github.com/picons/picons-source/files/1710340/rsi.default.svg_mcfly822.zip)

**Comment by Lexzie** [comment]
I see where I went wrong, when selecting the text and changing the style of the font nothing changed, so I moved on to the next font and so on...
But I had to press apply to change the style.

Even so, excellent work and I guess the  design of the logo did not cost a lot of money for RSI. 😆 
The letters of `INTERNATIONALE` are a little bigger then `RADIO SENEGAL`, is that intentional?
I don't see a difference in size in the original logo.
After resizing this is the result:
[radiosenegalinternationale.zip](https://github.com/picons/picons-source/files/1711779/radiosenegalinternationale.zip)

**Add to snp.index**
```
rsi=radiosenegalinternationale
```

**Add to srp.index**
```
1654_9EFC_7E_462BD5=radiosenegalinternationale
```

`donkey monkey got issued`
